services/tables/src/main/java/com/linkedin/openhouse/tables/api/spec/v0/request/CreateUpdateTableRequestBody.java

```diff
@@ -103,6 +104,7 @@ public class CreateUpdateTableRequestBody {
 
   @Schema(description = "The type of a table")
   @Valid
+  @Builder.Default
   private TableType tableType = TableType.PRIMARY_TABLE;
 
```

Comment:
can we do a log-screenshot check again at tableservice layer that verifies:
1) if request field is not provided by the client, springboot sets it as the respective default
2) if request field is provided by the client, springboot honors it
?

Reply:
Will do!

Reply:
Finally got around to this. Here are screenshots of 2 curl commands. One does not specify a table type and the other specifies a replica table type (i.e. not the default). The screenshots show the proper responses. 
<img width="1773" alt="Screenshot 2024-06-19 at 11 44 12 AM" src="https://github.com/linkedin/openhouse/assets/940916/d40ae14b-be73-4e0d-ac06-3f06fb047eec">
<img width="1555" alt="Screenshot 2024-06-19 at 11 46 03 AM" src="https://github.com/linkedin/openhouse/assets/940916/ec4fa4c3-f2ce-4237-a419-e2ace508a762">

Reply:
Also here are screenshots of the persisted metadata in the iceberg metadata file for the tables showing the proper table type.
<img width="815" alt="Screenshot 2024-06-19 at 3 55 18 PM" src="https://github.com/linkedin/openhouse/assets/940916/5d3e7ffe-09b4-476f-a2c1-64e17d0dfcf4">
<img width="880" alt="Screenshot 2024-06-19 at 3 56 42 PM" src="https://github.com/linkedin/openhouse/assets/940916/708d30ff-c507-4e71-a28a-1814c7bb98ed">